quipucords/api/insights_report/tests_insights_report.py

```diff
@@ -91,6 +91,25 @@ def setUp(self):
                         "rh_products_installed": ["RHEL", "EAP", "FUSE"],
                     }
                 ],
+                "system_profile": {
+                    "number_of_cpus": 1,
+                    "number_of_sockets": 1,
+                    "cores_per_socket": 1,
+                    "system_memory_bytes": 1041051648,
+                    "infrastructure_type": "virtual",
+                    "os_release": "Red Hat Enterprise Linux \
+                                Server release 5.9 (Tikanga)",
+                    "arch": "x86_64",
+                    "cloud_provider": "aws",
+                    "network_interfaces": [
+                        {
+                            "ipv4_addresses": ["10.10.181.116"],
+                            "ipv6_addresses": [],
+                            "name": "unknown",
+                            "mac_address": "00:50:56:9e:7b:19",
+                        }
+                    ],
+                },
```

Comment:
I don't get it. Which test is consuming this data? Is it testing the legacy and **soon to be removed** cached insights column?

Reply:
It is. No wonder system-profile was not there previously. I will remove it.